R/idr0021 - Segmentation.ipynb

```diff
@@ -0,0 +1,702 @@
+    "\n",
+    "As we go through the notebook you're expected to run the code blocks in order (unless explicitely mentioned otherwise).\n",
+    "\n",
+    "Here's an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "message(\"Just wait for a bit...\")\n",
+    "Sys.sleep(3)\n",
+    "message(\"Done.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can change the code and run the block again.\n",
+    "\n",
+    "**Exercise:** Change the message which is printed and run it again."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Let's get started..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load the necessary [romero-gateway](https://github.com/ome/rOMERO-gateway) and [EBImage](https://bioconductor.org/packages/release/bioc/html/EBImage.html) library. We're currently using romero-gateway 0.4.3, EBImage 4.13.0 with R 3.4.1 . "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "library(romero.gateway)\n",
+    "library(EBImage, warn.conflicts = FALSE)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log into the OMERO server"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "user_name = readline('Username: ')\n",
+    "user_password <- getPass::getPass('OMERO password: ')\n",
+    "server <- OMEROServer(host = 'outreach.openmicroscopy.org', username=user_name, password=user_password, port= as.integer(4064))\n",
+    "server <- connect(server)\n",
+    "paste('Successfully logged in as', server@user$getUserName())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load and display an image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load an image from the example dataset 'R-dataset' (these images are part of the original 'CDK5RAP2-C' dataset).\n",
+    "\n",
+    "**Exercise:** Go to the image 'siControl_N20_Cep215_I_20110411_Mon-1509_0_SIR_PRJ.dv' in your 'R-dataset', find the 'Image ID' and copy/paste it below. You can use a different image of the dataset if you like, but I recommend that one because it nicely shows the toroid centrioles."
```

Comment:
``I`` -> ``We`` :)

But maybe rather:

``Select the image 'siControl_N20_Cep215_I_20110411_Mon-1509_0_SIR_PRJ.dv' in the "R-dataset" in OMERO. From the right-hand pane, copy the ImageID. Paste it into the code cell below.

Reply:
Leave out the sentence aoubt ``You can use a different image if you like....`` It is too long. Nobody knows what the centrioles are at this stage. I guess they will understand that they can use another image. 

Possibly


``Exercise: Find in OMERO.web an ImageID of a different image and display it in R.``

Reply:
```
Found more than one class "Image" in cache; using the first, from namespace 'romero.gateway'
Also defined by ‘EBImage’

```

This is the output. Hum, not really nice.